packages/medusa/src/api/routes/admin/orders/list-orders.ts

```diff
@@ -214,7 +215,12 @@ export default async (req, res) => {
     data = orders.map((o) => pick(o, fields))
   }
 
-  res.json({ orders: data, count, offset: skip, limit: take })
+  res.json({
+    orders: cleanResponseData(data, req.allowedProperties),
```

Comment:
Everything should be allowed for the admin end point. I suspect that this end point is fixed just by this [Pr](https://github.com/medusajs/medusa/pull/3220) as if the user want only the id and no relation then he can provide fields=id and expand= wdy? I fill like we are using a side effect of a feature to do the cleanup

Reply:
Yes, you are right!